bazel/copts.bzl

```diff
@@ -10,18 +10,14 @@ copts_common = [
 def copts():
     return copts_common + select({
         "@bazel_tools//src/conditions:darwin": [
-            "-std=c++17",
             # We do not want to treat `syscall` warning as an error.
             # Future update of glog lib will fix this warning.
             "-Wno-error=deprecated-declarations",
```

Comment:
Let's follow up in a later PR and see if we can get rid of `copts.bzl` all together!